website/api/zkvm/optimization.md

```diff
@@ -447,9 +447,9 @@ cycle counts added.
 | OR rd,rs1,rs2       | Or                                 | rd ← ux(rs1) ∨ ux(rs2)                         | 2                                               |
 | AND rd,rs1,rs2      | And                                | rd ← ux(rs1) ∧ ux(rs2)                         | 2                                               |
 | MUL rd,rs1,rs2      | Multiply                           | rd ← ux(rs1) × ux(rs2)                         | 1                                               |
-| MULH rd,rs1,rs2     | Multiply High Signed Signed        | rd ← (sx(rs1) × sx(rs2)) » xlen                | 1                                               |
+| MULH rd,rs1,rs2     | Multiply High Signed               | rd ← (sx(rs1) × sx(rs2)) » xlen                | 1                                               |
```

Comment:
This is wrong, the instruction takes two operands, and is describing the signed-ness of each.